### DISCUSSION TITLE
Accessing env variables inside remix.config.js

### DISCUSSION THREAD

**Discussion by KasparRosin** [remix-run/remix#3808] [Q&A] [answered]
I noticed that process.env was available in routes function inside remix.config.js, but the env variables were not.
Is it a bug, is there a workaround?

Any help is appreciated.

**Comment by kiliman** [comment, marked as answer]
You need to load the `.env` variables using `dotenv`. 
```ts
// remix.config.js
require('dotevnv').config()
```
https://www.npmjs.com/package/dotenv
